workflows/prognostic_c48_run/runtime/metrics-schema.json

```diff
@@ -234,6 +234,31 @@
             "default": 0.0,
             "units": "kg/m^2/s"
         },
+        "storage_of_internal_energy_path_due_to_python": {
+            "type": "number",
+            "default": 0.0,
+            "units": "J/kg kg/m**2/s"
```

Comment:
Units typo?

Reply:
Ideally it would be written as W/m**2 (which this is equivalent to) but since the `monitor.compute_change` function is quite general it does not know to do this simplification. It just adds the kg/m**2/s units (mass integral+tendency) to whatever units the input has. We could add some units package but that seems like overkill.